sass/index.scss

```diff
@@ -435,6 +436,119 @@ video::-webkit-media-controls {
     }
 }
 
+// Prizes
+#prizes {
+    $tape-offset: 35px;  // how far into the card is the tape placed?
+    
+    @mixin tape-transform($rotation) {
+        transform: translate(-50%, -50%) rotate($rotation);
+        // this makes it so that we're positioning the tape by its center instead of its top-left, pre-rotation
+        // corner, which is significantly easier
+    }
+
+    #prizes-ricky {
+        // placeholder
+        width: 85%;
+        max-width: 980px;
+        height: auto;
+        background-color: blue;
+        margin: 0 auto;
+    }
+
+    #prizes-card {
+        background-color: white;
+        border: 1px solid $light-blue;
+        border-radius: 6px;
+        margin-top: -195px;  // need to overlap prizes ricky
+        padding: 75px 95px;
+        position: relative;
+        width: 100%;
+
+        .tape {
+            background-color: $beige;
+            height: 40px;
+            opacity: 25%;
+            position: absolute;
+            width: 165px;
+
+            &.top-left {
+                @include tape-transform(-45deg);
+                left: $tape-offset;
+                top: $tape-offset;
+            }
+
+            &.top-right {
+                @include tape-transform(45deg);
+                right: $tape-offset - 165px;
+                top: $tape-offset;
+            }
+
+            &.bottom-left {
+                @include tape-transform(45deg);
+                left: $tape-offset;
+                bottom: $tape-offset - 40px;
+            }
+
+            &.bottom-right {
+                @include tape-transform(-45deg);
+                right: $tape-offset - 165px;
+                bottom: $tape-offset - 40px;
+            }
+        }
+
+        & > .content {
+            align-items: flex-end;
+            column-gap: 50px;
+            display: flex;
+            flex-direction: row;
+            margin-bottom: 100px;
+
+            .graphic {
+                flex: 0 0 auto;
+            }
+
+            .text {
+                display: flex;
+                flex-direction: column;
+                row-gap: 20px;
+
+                .title {
+                    letter-spacing: 1.5rem;
+                    opacity: 50%;
```

Comment:
this is showing up as 1% on firefox for some reason:
<img width="1616" alt="CleanShot 2022-02-22 at 21 29 18@2x" src="https://user-images.githubusercontent.com/2782749/155252432-fe68abb3-3ead-4f7f-9b9a-6ac1ec509c60.png">

Reply:
This is not showing up in chrome for me, either; it's also set to 1% in my inspect.

Technically we should actually be using decimals instead of percent, since percent opacity doesn't work in most browsers ([mozilla](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity#browser_compatibility))